<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sklad</class>
 <widget class="QWidget" name="sklad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>915</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>915</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Учет и ремонт Транспортных средств</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Информация о неустановленных шинах</string>
   </property>
   <widget class="QLineEdit" name="searchQuerySklad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableSklad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>771</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="searchIconSklad">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>25</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>18</y>
     <width>101</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>Списать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Замена шин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRefresh">
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>428</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
